perf/load-incluster/helm/client/templates/client.yaml

```diff
@@ -0,0 +1,28 @@
+{{range $i := until (int .Values.entries)}}
+---
+apiVersion: apps/v1beta1
+kind: Deployment
+metadata:
+  name: fortio-cli-{{$i}}
+spec:
+  replicas: 1
+  template:
+    metadata:
+      labels:
+        app: fortio-cli-v{{$i}}
+    spec:
+      containers:
+      - name: fortio-cli
+        image: istio/fortio:latest
+        imagePullPolicy: IfNotPresent
+        args:
+          - load
+          - -t
+          - "0"
+          - -c
+          - "8"
+          - -qps
+          - "500"
+          - http://fortio-server-{{$i}}.default:8080/echo?size=128
```

Comment:
make namespace to torture unique.

Reply:
I think you can just leave off the namespace since they are all in the same namespace?